lib/resolver.content.js

```diff
@@ -133,13 +134,29 @@ export default class PodletClientContentResolver {
             outgoing.contentUri,
         );
 
+        let assetObjects;
+
         /** @type {import('./http.js').PodiumHttpClientRequestOptions} */
         const reqOptions = {
             rejectUnauthorized: outgoing.rejectUnauthorized,
             timeout: outgoing.timeout,
             method: 'GET',
             query: outgoing.reqOptions.query,
             headers,
+            onInfo: ({ statusCode, headers }) => {
+                if (statusCode === 103) {
+                    // send early hints
```

Comment:
> Early hints are received from the podlet, parsed and then resent to the browser as preload early hints

Is this where that's happening? Maybe expand on the comment and make that clearer, I missed that detail the first time around.

Reply:
might even consider making it a separate function, leaving more room for docs